--- build-deb-packages.sh       2023-03-31 17:05:29.144281993 +0000
+++ build-deb-packages-drv.sh   2023-03-31 17:20:39.950769640 +0000
@@ -80,14 +80,9 @@


 if [[ ${build_test_package} == 1 ]] && [ "X$CUDA" == "X" ]; then
-    echo "CUDA environment variable is not defined"; exit 1
+    echo "CUDA environment variable is not defined"; #exit 1
 fi

-NVCC=${CUDA}/bin/nvcc
-CUDA_VERSION=`$NVCC --version | grep release | sed 's/^.*release \([0-9]\+\.[0-9]\+\).*/\1/'`
-CUDA_MAJOR=`echo ${CUDA_VERSION} | cut -d "." -f 1`
-CUDA_MINOR=`echo ${CUDA_VERSION} | cut -d "." -f 2`
-
 echo "Building debian package for the gdrcopy library ..."

 ex cd ${SCRIPT_DIR_PATH}
@@ -159,6 +154,8 @@
 debuild_params+=" -us -uc"
 ex debuild ${debuild_params}

+echo "Force disable build_test_package..."
+build_test_package=0
 if [[ ${build_test_package} == 1 ]]; then
     echo
     echo "Building gdrcopy-tests package ..."
@@ -197,22 +194,22 @@
     ex dpkg-buildpackage -rfakeroot -d -F -us -uc
 fi

-echo
-echo "Building gdrcopy package ..."
-metadir=${tmpdir}/gdrcopy-${VERSION}
-ex mkdir -p ${metadir}
-ex cd ${TOP_DIR_PATH}
-ex cp -r packages/debian-meta ${metadir}/debian
-ex cp README.md ${metadir}/debian/README.Debian
-ex cp README.md ${metadir}/debian/README.source
-ex cd ${metadir}
-ex find . -type f -exec sed -i "s/@FULL_VERSION@/${FULL_VERSION}/g" {} +
-ex find . -type f -exec sed -i "s/@VERSION@/${VERSION}/g" {} +
-ex find . -type f -exec sed -i "s/@MODULE_LOCATION@/${MODULE_SUBDIR//\//\\/}/g" {} +
-ex cd ${tmpdir}
-ex tar czvf gdrcopy_${VERSION}.orig.tar.gz gdrcopy-${VERSION}
-cd ${metadir}
-ex debuild -us -uc
+#echo
+#echo "Building gdrcopy package ..."
+#metadir=${tmpdir}/gdrcopy-${VERSION}
+#ex mkdir -p ${metadir}
+#ex cd ${TOP_DIR_PATH}
+#ex cp -r packages/debian-meta ${metadir}/debian
+#ex cp README.md ${metadir}/debian/README.Debian
+#ex cp README.md ${metadir}/debian/README.source
+#ex cd ${metadir}
+#ex find . -type f -exec sed -i "s/@FULL_VERSION@/${FULL_VERSION}/g" {} +
+#ex find . -type f -exec sed -i "s/@VERSION@/${VERSION}/g" {} +
+#ex find . -type f -exec sed -i "s/@MODULE_LOCATION@/${MODULE_SUBDIR//\//\\/}/g" {} +
+#ex cd ${tmpdir}
+#ex tar czvf gdrcopy_${VERSION}.orig.tar.gz gdrcopy-${VERSION}
+#cd ${metadir}
+#ex debuild -us -uc

 echo
 echo "Copying *.deb and supplementary files to the current working directory ..."
